### ISSUE TITLE
Wrong camera layout for Station Square in SALVL

### ISSUE THREAD

**Issue by kellsnc** [X-Hax/sa_tools#272]
Opening any of the Station Square acts in SALVL (except the first and last) will open the wrong camera layout.

This is because the game has lookup tables to match an act ID to a SET/CAM ID. For most of the levels it's a direct match, *except* for Station Square cameras where each act ID maps to the following layout ID: < 0, 3, 1, 4, 2, 5 >.

FYI, the SET lookup table is at 0x7E5E98 and the CAM one is at 0x7E5F48. It's a series of eight 4-bit values per level. The input is the act ID, the output is the layout ID.

We could:
- Hardcode an exception for Station Square
- Provide some attribute in some ini file to define the layouts to load for each act

**Comment by MainMemory** [comment]
So wait, it uses a different CAM file than the SET file? I suppose we could add a `CAMName` field to the INI alongside the existing `SETName` field.

**Comment by kellsnc** [comment]
Yes, for some reason only the Station Square cameras don't match the act numbers, the set files match.

Act 0 = CAMSS00S
Act 1 = CAMSS03S
Act 2 = CAMSS01S
Act 3 = CAMSS04S
Act 4 = CAMSS02S
Act 5 = CAMSS05S
